bin/mastarm

```diff
@@ -9,7 +9,7 @@ const util = require('../lib/util')
 commander
   .version(require('../package.json').version)
   .option('-c, --config <path>', 'Path to configuration files.', path.join(process.cwd(), '/configurations/default'))
-  .option('-e, --env <environment>', 'Environment to use.', process.env.NODE_ENV || 'development')
+  .option('-e, --env <environment>', 'Environment to use.')
```

Comment:
See https://github.com/conveyal/scenario-editor/blob/dev/package.json#L15.  I think this approach is good too for the added flexibility of setting the environment from other places.